DigitalOffice/Services/ProjectService/Models/Project.cs

```diff
@@ -0,0 +1,12 @@
+﻿using ProjectService.Database.Entities;
+using System.Collections.Generic;
+
+namespace ProjectService.Models
+{
+    public class Project
```

Comment:
Judging by the SwagerHub, the model also should contain fields : Id, Description, DepartmentId.
Also, your model contains collections of database entities, which is bad for security.
Model should contain sequences of Managers' and Workers' GUID.

Reply:
About the fields of the model: we discussed it with Spartak and he said it's enough for now.
Project model fields(Managers and Workers Ids) are fixed. Now they're IEnumerable<Guid>.

Reply:
Fixed

Reply:
Idk, I were said it's enough
![image](https://user-images.githubusercontent.com/59228285/89015470-bf96cf00-d31f-11ea-90b8-c41a0880a365.png)